### PR TITLE
removed illustrationOutside

### PR DESCRIPTION
**Pull request by iFun**
The design no longer require illustration-outside from the list so lets remove the attribute. In the next stories that prepare the lists for drag n drop will remove this functionality and I want to make sure no one starts using it before we get to that. So this story is a quick disable this attribute from doing anything.

[US115330](https://rally1.rallydev.com/#/357252966636d/custom/367300408400?qdp=%2Fdetail%2Fuserstory%2F381655117040)

### CODE REVIEW

File: components/list/test/list.visual-diff.js [hunk 1]
```diff
@@ -29,7 +29,6 @@ describe('d2l-list', () => {
 		]},
 		{ category: 'illustration', tests: [
 			{ name: 'default', selector: '#illustration' },
-			{ name: 'outside', selector: '#illustrationOutside' }
```
Comment (dlockhart) [inline components/list/test/list.visual-diff.js:32]: You'll need to delete the actual golden screenshot image as well.

File: components/list/list-item.js [hunk 2]
```diff
@@ -27,7 +27,6 @@ class ListItem extends RtlMixin(LitElement) {
 			breakpoints: { type: Array },
 			disabled: {type: Boolean },
 			href: { type: String },
-			illustrationOutside: { type: Boolean, attribute: 'illustration-outside' },
```
Comment (dbatiste) [inline components/list/list-item.js:30]: There's a bunch of CSS selectors that hook into this attribute.  ex.  `:host([illustration-outside]) ...` that should also be removed.

Comment (m6jones) [reply components/list/list-item.js:30]: Ah, you bet me to it. See my comment for the CSS to be removed.

**Comment by iFun** [comment]
> I saw a couple more places to delete code:
> 
> ```
> :host([illustration-outside]) .d2l-list-item-content-flex {
> 				padding: 0.55rem 0;					padding: 0.55rem 0;
> 			}				}
> 			:host([illustration-outside]) .d2l-list-item-container ::slotted([slot="illustration"]),				:host([illustration-outside]) .d2l-list-item-container ::slotted([slot="illustration"]),
> 			:host([illustration-outside]) .d2l-list-item-content-flex ::slotted([slot="illustration"]) {				:host([illustration-outside]) .d2l-list-item-content-flex ::slotted([slot="illustration"]) {
> 				margin-bottom: 0.7rem;					margin-bottom: 0.7rem;
> 				margin-top: 0.7rem;					margin-top: 0.7rem;
> 			}				}
> 			:host([illustration-outside]) input[type="checkbox"].d2l-input-checkbox {				:host([illustration-outside]) input[type="checkbox"].d2l-input-checkbox {
> 				margin-bottom: 1.15rem;					margin-bottom: 1.15rem;
> 				margin-top: 1.15rem;					margin-top: 1.15rem;
> 			}				}
> ```
> 
> Sorry, wish I could just comment on non changes :(

thank you working on delete these code

File: components/list/list-item.js [hunk 3]
```diff
@@ -269,21 +253,20 @@ class ListItem extends RtlMixin(LitElement) {
...
 				${label}
-				${this.illustrationOutside ? beforeContent : null}
 				${link}
 				<div id="${this._contentId}"
 					class="d2l-list-item-content"
 					?extend-separators="${this._extendSeparators}"
 					separators="${ifDefined(this._separators)}">
 					<div class="d2l-list-item-content-flex">
-						${!this.illustrationOutside ? beforeContent : null}
+						${beforeContent}
```
Comment (zinabat) [inline components/list/list-item.js:269]: Shouldn't this one also be removed entirely?

**Pull request merged**